openedx/features/subscriptions/api/v1/tests/test_models.py

```diff
@@ -0,0 +1,141 @@
+from datetime import date, datetime, timedelta
+import ddt
+import pytest
+
+from openedx.features.subscriptions.models import UserSubscription
+from openedx.features.subscriptions.api.v1.tests.factories import UserSubscriptionFactory
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+pytestmark = pytest.mark.django_db
+
+LIMITED_ACCESS = 'limited-access'
```

Comment:
Remove these static variables and use `UserSubscription` constants, like `UserSubscription.LIMITED_ACCESS`

Update everywhere

Reply:
There was an issue with DB access while I was writing tests. I thought it was because of using model constants outside of test functions but turned out to be something else. removed these.